### PR TITLE
mirrors: Add KZ mirror.

### PR DESCRIPTION
**Pull request by MrLimb**

### CODE REVIEW

**Comment by ericonr** [comment]
Nice, thanks! Could you reword the commit message, please? `mirros: Add KZ mirror.` is good.

File: src/xbps/repositories/mirrors/index.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ sub-repository.
 | <https://ftp.gnome.org/mirror/voidlinux/>          | EU: Sweden        |
 | <https://ftp.lysator.liu.se/pub/voidlinux/>        | EU: Sweden        |
 | <https://ftp.sunet.se/mirror/voidlinux/>           | EU: Sweden        |
+| <https://mirror.ps.kz/voidlinux/>                  | KZ: Kazakhstan    |
```
Comment (ericonr) [inline src/xbps/repositories/mirrors/index.md:50]: You probably want to use `Asia: Kazakhstan`, instead. Unless there is a city in Kazakhstan named Kazakhstan? Even then, I would use `Asia: Kazakhstan, KZ`

**Comment by the-maldridge** [comment]
spellcheck is a wonderful thing...

File: src/xbps/repositories/mirrors/index.md (hunk 1 above)
Comment (MrLimb) [reply src/xbps/repositories/mirrors/index.md]: Thanks, I changed it to `Asia: Kazakhstan`, because a longer string may break the list formatting...

Comment (ericonr) [reply src/xbps/repositories/mirrors/index.md]: If you'd like to increase the column width, I don't have any issue with it!

File: src/xbps/repositories/mirrors/index.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ sub-repository.
 | <https://ftp.gnome.org/mirror/voidlinux/>          | EU: Sweden        |
 | <https://ftp.lysator.liu.se/pub/voidlinux/>        | EU: Sweden        |
 | <https://ftp.sunet.se/mirror/voidlinux/>           | EU: Sweden        |
+| <https://mirror.ps.kz/voidlinux/>                  | Asia: Kazakhstan  |
```
Comment (ericonr) [inline src/xbps/repositories/mirrors/index.md:50]: Insert it into the list using alphabetical order, please. And please squash your commits after you're done c:

Comment (MrLimb) [reply src/xbps/repositories/mirrors/index.md]: I think, I should get enough sleep. с: I fixed everything, added a city and squash commits. Sorry for this awful chaos. Thanks for the help!

Comment (ericonr) [reply src/xbps/repositories/mirrors/index.md]: No problem, thanks for the PR!

**Pull request merged**